### PR TITLE
feat: setup prisma, connect it to mongo & prepare user schema

### PR DESCRIPTION
**Pull request by motaz99**

### CODE REVIEW

**Comment by motaz99** [comment]
https://vimeo.com/1030298945/4d6a0eaf20?share=copy

File: prisma/schema.prisma
Comment (MohammedAlabd) [inline prisma/schema.prisma:1]: are you planing to host this on vercel?
if yes, you should take a look at this 

https://www.prisma.io/docs/orm/prisma-client/deployment/serverless/deploy-to-vercel

Comment (MohammedAlabd) [reply prisma/schema.prisma:1]: and this one
https://www.mongodb.com/docs/atlas/reference/partner-integrations/vercel/

**Review by MohammedAlabd — APPROVED** [review]
LGTM, just check my comments

**Pull request merged**